.github/workflows/ci.yml

```diff
@@ -48,14 +48,59 @@ jobs:
       - name: Run database migrations
         run: mix ecto.migrate --prefix realtime
 
-      # - name: Compile with warnings
-      #   run: mix compile --warnings-as-errors
+      - name: Run tests
+        run: mix test
+  mix-format:
+    name: Format ${{ matrix.otp }} / Elixir ${{ matrix.elixir }}
 
-      # - name: Run formatter
-      #   run: mix format --check-formatted
+    defaults:
+      run:
+        working-directory: ./server
+
+    strategy:
+      matrix:
+        platform: [ubuntu-18.04]
+        otp: [23.x]
+        elixir: [1.11.x]
 
-      # - name: Run Credo
-      #   run: mix credo
+    runs-on: ubuntu-18.04
```

Comment:
Is there a reason to be on 18 and not the latest LTS?

Reply:
Not quite -- will modify to 20.04 instead. Thanks for the catch!